### ISSUE TITLE
poetry issue: can't install or build wheel on linux

### ISSUE THREAD

**Issue by RemiTorracinta** [siddhantgoel/streaming-form-data#33]
Command to reproduce:
```bash
docker run --entrypoint=pip3 python:3.6 install "streaming-form-data==1.5.1"
```
Error message / stack trace:

```
  ERROR: Command errored out with exit status 1:
   command: /usr/local/bin/python /usr/local/lib/python3.6/site-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /tmp/tmpv61zx81r
       cwd: /tmp/pip-wheel-j1vdg9g0/streaming-form-data
  Complete output (14 lines):
  Traceback (most recent call last):
    File "/usr/local/lib/python3.6/site-packages/pip/_vendor/pep517/_in_process.py", line 257, in <module>
      main()
    File "/usr/local/lib/python3.6/site-packages/pip/_vendor/pep517/_in_process.py", line 240, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
    File "/usr/local/lib/python3.6/site-packages/pip/_vendor/pep517/_in_process.py", line 91, in get_requires_for_build_wheel
      return hook(config_settings)
    File "/tmp/pip-build-env-ztxe98dq/overlay/lib/python3.6/site-packages/poetry/masonry/api.py", line 25, in get_requires_for_build_wheel
      poetry = Factory().create_poetry(Path("."))
    File "/tmp/pip-build-env-ztxe98dq/overlay/lib/python3.6/site-packages/poetry/factory.py", line 54, in create_poetry
      raise RuntimeError("The Poetry configuration is invalid:\n" + message)
  RuntimeError: The Poetry configuration is invalid:
    - Additional properties are not allowed ('url' was unexpected)

```

**Comment by siddhantgoel** [comment]
What version of Poetry are you using?

**Comment by RemiTorracinta** [comment]
1.0.0 last I checked. Guess a major release that came out 3 days ago is
likely to cause a problem.

Perhaps you can set up a version lock on poetry? Don’t know too much about
it but as far as I can tell it’s installed for the user when they try to
install, as it’s needed for building.

As you can see from my command on a fresh docker container, poetry was
never specified or preinstalled.

**Comment by siddhantgoel** [comment]
I just pushed a fix removing the `url` property. I guess they removed it in 1.0.0. Should work fine now.

**Comment by EccoTheFlintstone** [comment]
would it be possible to bump the version and publish the new/fixed one on pypi?

thanks

**Comment by siddhantgoel** [comment]
Sure. Just published `v1.5.2`.

**Comment by EccoTheFlintstone** [comment]
thanks a lot
